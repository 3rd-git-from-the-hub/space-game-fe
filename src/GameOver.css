.game-over-text {
    position: absolute;
    top: 20px;
    color: white;
}

.game-over-link {
    color: teal;
    margin: 50px;
}