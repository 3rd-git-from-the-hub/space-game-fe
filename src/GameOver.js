import React, { Component } from 'react'

export default class GameOver extends Component {
    render() {
        return (
            <div>
                <p>GAME OVER</p>
            </div>
        )
    }
}
