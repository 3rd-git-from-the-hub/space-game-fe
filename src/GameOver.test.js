import React from 'react';
import { shallow } from 'enzyme';
import GameOver from './GameOver.js';

test('renders gameover correctly no props', () => {
  const wrapper = shallow(<GameOver />);
  expect(wrapper).toMatchSnapshot();
});

test('renders gameover correctly with props', () => {
    const wrapper = shallow(<GameOver 
        hull={0}
        fuel={5}
    />);
    expect(wrapper).toMatchSnapshot();
  });