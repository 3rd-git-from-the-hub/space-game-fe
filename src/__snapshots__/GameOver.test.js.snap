// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders gameover correctly no props 1`] = `
<div>
  <p />
  <p>
    You have won 
     times!
  </p>
</div>
`;

exports[`renders gameover correctly with props 1`] = `
<div>
  <p>
    The people of Earth realize that something must have gone wrong. They mourne your loss and raise a memorial in your name. You provide the courage for future Travelers to take the journey.
  </p>
  <p>
    You have won 
     times!
  </p>
</div>
`;
